models/fellow.py

```diff
@@ -4,25 +4,29 @@
 from models.person import Person
 
 class Fellow(Person):
-    """ Create a class Fellow which inherits from the Person clss"""
+	""" Create a class Fellow which inherits from the Person clss"""
 
-    fellow_data = []
+	fellow_data = []
 
-    def __init__(self, first_name, last_name):
-        """Initializes a new fellow and assigns properties of the fellow"""
-        super(Fellow, self).__init__(first_name, last_name)
-        self.person_type = 'fellow'
-        self.fellow_id = self.get_fellow_id()
-        Fellow.fellow_data.append(self.fellow_id)
+	def __init__(self, first_name, last_name):
+		"""Initializes a new fellow and assigns properties of the fellow"""
+		super(Fellow, self).__init__(first_name, last_name)
+		self.person_type = 'FELLOW'
+		self.fellow_id = self.get_fellow_id()
+		Fellow.fellow_data.append(self.fellow_id)
+		self.office_allocated = False
+		self.office=''
+		self.livingspace_allocated = False
+		self.livingspace = ''
 
-    def get_fellow_id(self):
-        """Returns unique id for Fellow by generating random string"""
-        id = 'F' + ''.join(random.choice(string.ascii_uppercase + string.digits) for _ in range(7))
+	def get_fellow_id(self):
+		"""Returns unique id for Fellow by generating random string"""
+		id = 'F' + ''.join(random.choice(string.ascii_uppercase + string.digits) for _ in range(7))
```

Comment:
`id` is a built in Python keyword and shouldn't be used as a variable name.

`fellow_id` is a more suitable name.